tests/test_adapter.py

```diff
@@ -681,39 +731,39 @@ def assert_arangodb_data(
 
         assert collection.count() == num_nodes
 
-        # TODO: Remove str restriction
-        has_node_feature_matrix = "x" in node_data and type(meta.get("x", "x")) is str
-        has_node_target_label = (
-            num_nodes == len(node_data.get("y", [])) and type(meta.get("y", "y")) is str
-        )
+        # # TODO: Remove str restriction
+        # has_node_feature_matrix = "x" in node_data and type(meta.get("x", "x")) is str
+        # has_node_target_label = (
+        #     num_nodes == len(node_data.get("y", [])) and type(meta.get("y", "y")) is str
```

Comment:
todo: re-enable feature/label in for `assert_arangodb_data` (this components needs to be refactored, as it is now out of date due to the new supported values in the adapter `metagraph`)